pkg/server/routes.go

```diff
@@ -39,8 +39,8 @@ func (srv *RegistrationServer) SetupRoutes() error {
 		unsecuredV1 := srv.router.Group("/api/v1")
 		unsecuredV1.GET("/health", healthCheckCtrl.GetHandler)
 		unsecuredV1.GET("/authconfig", authConfigCtrl.GetHandler)
-		unsecuredV1.GET("/woopra-domain", woopraCtrl.GetWoopraDomain)
-		unsecuredV1.GET("/segment-write-key", woopraCtrl.GetSegmentWriteKey)
+		unsecuredV1.GET("/woopra-domain", woopraCtrl.GetDevSpacesWoopraDomain)
+		unsecuredV1.GET("/segment-write-key", woopraCtrl.GetSandboxSegmentWriteKey)
```

Comment:
We should add an endpoint for GetDevSpacesSegmentWriteKey too

Reply:
done in [3befcfd](https://github.com/codeready-toolchain/registration-service/pull/323/commits/3befcfd9dcd0c7ea9c2515921c69413c5ef1f927)